packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -92,9 +92,19 @@ export interface VirtualNodeListener {
   readonly portMapping?: PortMapping;
 
   /**
-   * Array fo HealthCheckProps for the node(s)
+   * Array of HealthCheckProps for the node(s)
```

Comment:
This is also not an array, but a single value 🤪 . Should we update that too, while we're here?